tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py36,static,docs
+envlist = py27,py36,static,docs,py38
```

I suggest to change py36 into py38 instead (to make it clear we don't want to support py36).

```suggestion
envlist = py27,py38,static,docs
```